src/neo/SmartContract/ApplicationEngine.Crypto.cs

```diff
@@ -68,19 +68,20 @@ private bool VerifyWithECDsa(StackItem item, byte[] pubkey, byte[] signature, EC
             }
         }
 
-        protected internal bool CheckMultisigWithECDsaSecp256r1(StackItem item, byte[][] pubkeys, byte[][] signatures)
+        protected internal bool CheckMultisigWithECDsaSecp256r1(StackItem item, byte[][] pubkeys, int m, byte[][] signatures)
```

Comment:
And this is purely an `ApplicationEngine.Convert` problem to me. The stack contains everything needed when doing `Neo_Crypto_CheckMultisigWithECDsaSecp256r1` call, it's just that this data gets translated into two arrays.